src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -67,21 +67,34 @@ def results_columns():
 def phenotype_database():
   """
     Phenotype Database table (non-bulk)
-  """
+  """  
   # Get the list of traits for non-bulk files
   try:
-    traits_non_bulk = query_phenotype_metadata()
+    page = request.args.get('page', 1, type=int)
+    per_page = 10
```

Comment:
Should the `per_page` value be stored as part of a config somewhere?  I'm actually not sure what the best place is, since you could make an argument it's a "front-end" value or a "back-end" value... maybe a question for @r-vieira?

(For now I think a constant val in-line is fine.)